### PR TITLE
Style/button styling

### PR DESCRIPTION
**Pull request by georgekwan**
## Why was this PR opened?
Change auth screen to use the same button as the rest of the screens
Change the button and text colour settings to local

### CODE REVIEW

**Comment by yanliu1111** [comment]
I like your changes, and reviewed all code and understand the logic. I approve this, but probably wait Tim or Maggie double confirm (because it is related to their code changes). 

Thank you for this refactor. I like all organized : )

**Comment by georgekwan** [comment]
> I like your changes, and reviewed all code and understand the logic. I approve this, but probably wait Tim or Maggie double confirm (because it is related to their code changes).
> 
> Thank you for this refactor. I like all organized : ) "BigButton" interesting name😏?

Yan, could you approve it?
I did those changes while I was screen-sharing with Maggie and Tim.

**Review by yanliu1111 — APPROVED** [review]

**Comment by maggiechew** [comment]
@georgekwan I'm seeing there's a conflict with package-lock.json, you might want to look into that. I tried running your branch and ran into some really interesting errors with react-native-keyboard-aware-scroll-view; I had to uninstall and reinstall it to get it working, no idea why.

File: apps/Learning-Buddy/screens/SignUpScreen.jsx [hunk 1]
```diff
@@ -97,7 +97,9 @@ export const SignUpScreen = ({ navigation }) => {
           />
         </View>
         {/* This button will need to pass values to auth process in future iterations */}
```
Comment (maggiechew) [inline apps/Learning-Buddy/screens/SignUpScreen.jsx:99]: Would you be so kind as to remove this comment before you pull? This is no longer required. :)

**Review by maggiechew — APPROVED** [review]
Looks good, just left a few comments about the conflict in package-lock.json and the comment that you could remove, but overall looks good.

File: apps/Learning-Buddy/screens/SignUpScreen.jsx (hunk 1 above)
Comment (georgekwan) [reply apps/Learning-Buddy/screens/SignUpScreen.jsx]: It is done

**Pull request merged**